### DISCUSSION TITLE
Custom characters in VIC bank 0

### DISCUSSION THREAD

**Discussion by IanWitham** [jkotlinski/durexforth#522] [Q&A] [answered]
Something I'd like to do is create custom characters in VIC bank 0, but I haven't been able to find spare RAM to do so in this region. Can anyone think of a way to do this?
I'm able to create custom characters in a different bank of course, but I'd like to be able to use the default screen mem at $0400. As far as I'm aware this is the only location to work with the "basic-like" printing routines.

**Comment by IanWitham** [comment, marked as answer]
I think I might have found a way. If I drop the v editor, that brings `here` back to $2747. Then I can put some characters at $2800 and put the rest of my program after that. Works in theory I think... now to try it in practice.
It also means I need to set up a VSCode to D64 to DurexForth workflow which I've been putting off doing 😁

**Reply by Whammo** [reply]
Calling ___modules___ will get you deeper in bank 0, but might be tricky.

**Comment by Whammo** [comment]
Issue ---modules--- to unload all modules, or ---editor--- to unload the editor only.
